components/update-profile-modal.js

```diff
@@ -0,0 +1,67 @@
+import MenteeForm from '../components/mentee-form';
+import MentorForm from '../components/mentor-form';
```

Comment:
It looks like you have a duplicate import here. You can reference an import multiple times within the same file without having to import it, so perhaps we can remove it?

Reply:
Hi Ellie, Thank you for the refactoring tips!  I'd love to incorporate all of these changes soon.  I'll respond to this one first because it's the easiest:  there is a mentEE form and a different mentOR form, so they are not duplicates. Is that what you meant?